libs/atqr/domain/tsconfig.spec.json

```diff
@@ -15,5 +15,5 @@
     "**/*.test.jsx",
     "**/*.spec.jsx",
     "**/*.d.ts"
-  ]
+, "src/lib/__mocks__/challenge-entity-mock.ts"  ]
```

Comment:
As the need for a challenge entity mock has already been discussed and agreed to not be needed ill ask you to remove this line from the compiler's include statement.